src/api/app/views/webui/request/_breadcrumb_items.html.haml

```diff
@@ -4,7 +4,7 @@
       Requests
   - else
     %li.breadcrumb-item
-      = link_to('Requests', requests_path)
```

Comment:
It wouldn't hurt to use `my_requests_path` instead

Reply:
Yeah, I guess that would be better... Changed.

Reply:
I modified it again to have the same filters as in the link of the left panel. Please take a look again.